moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/bullet_utils.h

```diff
@@ -99,17 +137,22 @@ inline btTransform convertEigenToBt(const Eigen::Isometry3d& t)
 
 /** @brief Tesseract bullet collision object.
  *
- *  A wrapper around bullet's collision object which contains specific information related to bullet */
+ *  A wrapper around bullet's collision object which contains specific information related to bullet. One of the main
+ *  differences is that a bullet collision object has a single world transformation and all shapes have transformation
+ *  relative to this world transform. The default is a active object and active objects are checked against active
```

Comment:
```suggestion
 *  relative to this world transform. The default is an active object and active objects are checked against active
```
Also, the default what? The default category?

Reply:
the default collision object category (we have active and static) and this defines the `m_collisionFilterGroup` and `m_collisionFilterMask`.